build-configs/integreat/index.ts

```diff
@@ -9,6 +9,13 @@ import mainImprint from './mainImprint'
 import { lightTheme } from './theme'
 
 const APPLE_APP_ID = '1072353915'
+const cityNotCooperatingTemplate =
```

Comment:
Could we move this to a separate file? I think it would also be nice if we did not have to duplicate it for integreat-test-cms or other build configs